src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs

```diff
@@ -778,6 +787,14 @@ void BuildDependencies(XmlNodeList dependencies)
                                     Type = type
                                 };
 
+                                if (string.IsNullOrEmpty(dependencyDetail.Version) ||
+                                    string.IsNullOrEmpty(dependencyDetail.Name) ||
+                                    string.IsNullOrEmpty(dependencyDetail.Commit) ||
+                                    string.IsNullOrEmpty(dependencyDetail.RepoUri))
+                                {
+                                    throw new DarcException("Either the name, version, commit or repo uri of a dependency was empty.");
+                                }
```

Comment:
We should just return it as is in this case, as we have been.  Otherwise we also have no way of getting all the existing dependencies to even update them and make them correct.